pingcap/tidb-binlog
# Skip the tiflash job.

### What problem does this PR solve? 
Fix abort when there'a `model.ActionUpdateTiFlashReplicaStatus` job caused the job.Query is empty:
https://github.com/pingcap/tidb-binlog/blob/cd9a4d17e1fcedf1b8c96873a5ca0b37be62d489/drainer/schema.go#L280

```
[ddl job sql miss]ID:85, Type:update tiflash replica status, State:synced, SchemaState:public, SchemaID:45, TableID:62, RowCount:0, ArgLen:0, start time: 2020-04-01 10:52:07.845 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0"] [errorVerbose="[ddl job sql miss]I      D:85, Type:update tiflash replica status, State:synced, SchemaState:public, SchemaID:45, TableID:62, RowCount:0, ArgLen:0, start time: 2020-04-01 10:52:07.845 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0\ngithub.com/pingcap/tidb-binlog/drainer.(*Schema).handl      eDDL\n\t/home/jenkins/agent/workspace/release_tidb_4.0/go/src/github.com/pingcap/tidb-binlo
```

### What is changed and how it works?
Skip the tiflash job.

### Check List 

Tests 

 
 - Manual test (add detailed scripts or steps below)
test:
1. create table t(id int);
2. ALTER TABLE t SET TIFLASH REPLICA 3 LOCATION LABELS "rack", "host", "abc";            // model.ActionSetTiFlashReplica
3. curl -X POST -d '{"id":45,"region_count":3,"flash_region_count":3}' https://127.0.0.1:10080/tiflash/replica  // model.ActionUpdateTiFlashReplicaStatus
4. performance write on table t and check can still replicate.

ref: https://github.com/pingcap/tidb/pull/12453

 - No code




Related changes

 - Need to cherry-pick to the release branch
 
 - Need to be included in the release note
    * Skip TiFlash relate DDL job to avoid replication abort.
